### ISSUE TITLE
FIXED: Game window does not highlight checker (man) move/attack positions, if forward positions are blocked

### ISSUE THREAD

**Issue by Quadprox** [Quadprox/zen-checkers-king#12]
Game window does not highlight move/attack positions, if forward positions are blocked:

White checker:
> ![image](https://user-images.githubusercontent.com/41553031/59041260-d328e000-88aa-11e9-97fd-f5b956c71276.png)
> ![image](https://user-images.githubusercontent.com/41553031/59041268-d8862a80-88aa-11e9-8cbe-2ba3a4d7bdd8.png)

Black checker:
> ![image](https://user-images.githubusercontent.com/41553031/59041326-f3589f00-88aa-11e9-893d-f76b27fad5d9.png)
> ![image](https://user-images.githubusercontent.com/41553031/59041335-f784bc80-88aa-11e9-9b4d-d6d07eabb4e7.png)

**Comment by Quadprox** [comment]
Queen-type checkers' move/attack positions are rendered properly:

White queen checker:
> ![image](https://user-images.githubusercontent.com/41553031/59041566-70841400-88ab-11e9-8127-aaaac0cb58b2.png)

Black queen checker:
> ![image](https://user-images.githubusercontent.com/41553031/59041522-5ba78080-88ab-11e9-8b7f-771696d3db3e.png)

**Comment by Quadprox** [comment]
> ![image](https://user-images.githubusercontent.com/41553031/59041827-fa33e180-88ab-11e9-8c6c-dd87454bac7f.png)

Selected white checker with only one forward position blocked:
```
White checker at D3 
moves: [[4, 3], [4, 5]] 
kill: []
```

> ![image](https://user-images.githubusercontent.com/41553031/59041915-1fc0eb00-88ac-11e9-922d-c204c86fab0d.png)

Selected white checker with both forward position blocked:
```
White checker at D3 
moves: [[5, 2], [5, 6]] 
kill: [[5, 2], [5, 6]]
```

**Comment by Quadprox** [comment]
Issue fixed with pull-request #14.
